diff --git a/score/mw/com/example/ipc_bridge/BUILD b/score/mw/com/example/ipc_bridge/BUILD
index 525969e..19e0fc3 100644
--- a/score/mw/com/example/ipc_bridge/BUILD
+++ b/score/mw/com/example/ipc_bridge/BUILD
@@ -31,6 +31,7 @@ cc_binary(
         "@score_baselibs//score/language/futurecpp",
         "@score_logging//score/mw/log",
     ],
+    visibility = ["//visibility:public"],
 )
 
 cc_library(
