diff --git a/MODULE.bazel b/MODULE.bazel
index 7b6f319e..b4c77340 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -98,6 +98,13 @@ deb(
     visibility = ["//visibility:public"],
 )
 
+deb(
+    name = "acl-deb-aarch64",
+    build = "//third_party/acl:acl.BUILD",
+    urls = ["https://ports.ubuntu.com/pool/main/a/acl/libacl1-dev_2.3.1-1_arm64.deb"],
+    visibility = ["//visibility:public"],
+)
+
 deb(
     name = "valgrind-deb",
     build = "//third_party/valgrind:valgrind.BUILD",
@@ -118,6 +125,19 @@ deb(
     visibility = ["//visibility:public"],
 )
 
+deb(
+    name = "libcap2-deb-aarch64",
+    urls = ["https://ports.ubuntu.com/pool/main/libc/libcap2/libcap2_2.25-1.2_arm64.deb"],
+    visibility = ["//visibility:public"],
+)
+
+deb(
+    name = "libcap2-dev-deb-aarch64",
+    build = "//third_party/libcap2:libcap2.BUILD",
+    urls = ["https://ports.ubuntu.com/pool/main/libc/libcap2/libcap-dev_2.25-1.2_arm64.deb"],
+    visibility = ["//visibility:public"],
+)
+
 deb(
     name = "libseccomp2-deb",
     urls = ["https://archive.ubuntu.com/ubuntu/pool/main/libs/libseccomp/libseccomp2_2.5.1-1ubuntu1~18.04.2_amd64.deb"],
@@ -131,6 +151,19 @@ deb(
     visibility = ["//visibility:public"],
 )
 
+deb(
+    name = "libseccomp2-deb-aarch64",
+    urls = ["https://ports.ubuntu.com/pool/main/libs/libseccomp/libseccomp2_2.5.3-2ubuntu2_arm64.deb"],
+    visibility = ["//visibility:public"],
+)
+
+deb(
+    name = "libseccomp2-dev-deb-aarch64",
+    build = "//third_party/libseccomp2:libseccomp2.BUILD",
+    urls = ["https://ports.ubuntu.com/pool/main/libs/libseccomp/libseccomp-dev_2.5.3-2ubuntu2_arm64.deb"],
+    visibility = ["//visibility:public"],
+)
+
 bazel_dep(name = "nlohmann_json", version = "3.11.3")
 bazel_dep(name = "bazel_skylib", version = "1.8.2")
 
diff --git a/score/language/safecpp/test/test_std_arithmetic_types.cpp b/score/language/safecpp/test/test_std_arithmetic_types.cpp
index b09415b4..5c4ed8a5 100644
--- a/score/language/safecpp/test/test_std_arithmetic_types.cpp
+++ b/score/language/safecpp/test/test_std_arithmetic_types.cpp
@@ -46,18 +46,6 @@ TEST(StdFloatTypes, IECConformance)
 
 TEST(StdIntegralTypes, signess_and_bytes_number)
 {
-// Test identifies implementation defined behavior.
-// As the standard does not specify if plain char is signed or unsigned.
-// char on QNX is considered as unsigned char and on GCC and Clang is considered as signed.
-#if defined(__QNX__)
-    // As the signess of char depends on using qemu or the real hardware, this line got commented until we decide
-    // How to move forward in this ticket Ticket-141110
-    // static_assert(std::numeric_limits<char>::is_signed == false,
-    //               "it is expected a 'char' to be unsigned in a QNX environment");
-#else
-    static_assert(std::numeric_limits<char>::is_signed == true,
-                  "it is expected a 'char' to be signed in a linux environment");
-#endif
     // Test identifies number of bytes
     static_assert(sizeof(char) == 1, "it is expected a char bytes' number to be 1");
 
diff --git a/third_party/acl/BUILD b/third_party/acl/BUILD
index b71d5df1..da079950 100644
--- a/third_party/acl/BUILD
+++ b/third_party/acl/BUILD
@@ -1,5 +1,8 @@
 alias(
     name = "acl",
-    actual = "@acl-deb//:acl",
+    actual = select({
+        "@platforms//cpu:aarch64": "@acl-deb-aarch64//:acl",
+        "//conditions:default": "@acl-deb//:acl",
+    }),
     visibility = ["//visibility:public"],
 )
diff --git a/third_party/acl/acl.BUILD b/third_party/acl/acl.BUILD
index 718726c6..b695a6ac 100644
--- a/third_party/acl/acl.BUILD
+++ b/third_party/acl/acl.BUILD
@@ -1,8 +1,9 @@
 cc_library(
     name = "acl",
-    srcs = [
-        "usr/lib/libacl.a",
-    ],
+    srcs = select({
+        "@platforms//cpu:aarch64": ["usr/lib/aarch64-linux-gnu/libacl.a"],
+        "//conditions:default": ["usr/lib/libacl.a"],
+    }),
     hdrs = [
         "usr/include/acl/libacl.h",
         "usr/include/sys/acl.h",
diff --git a/third_party/libcap2/BUILD b/third_party/libcap2/BUILD
index c74930f1..4704ee73 100644
--- a/third_party/libcap2/BUILD
+++ b/third_party/libcap2/BUILD
@@ -1,5 +1,8 @@
 alias(
     name = "libcap2",
-    actual = "@libcap2-dev-deb//:libcap2",
+    actual = select({
+        "@platforms//cpu:aarch64": "@libcap2-dev-deb-aarch64//:libcap2",
+        "//conditions:default": "@libcap2-dev-deb//:libcap2",
+    }),
     visibility = ["//visibility:public"],
 )
diff --git a/third_party/libcap2/libcap2.BUILD b/third_party/libcap2/libcap2.BUILD
index 05f504a6..489cefb4 100644
--- a/third_party/libcap2/libcap2.BUILD
+++ b/third_party/libcap2/libcap2.BUILD
@@ -1,9 +1,15 @@
 cc_library(
     name = "libcap2",
-    srcs = [
-        "@libcap2-deb//:lib/x86_64-linux-gnu/libcap.so.2",
-        "@libcap2-deb//:lib/x86_64-linux-gnu/libcap.so.2.25",
-    ],
+    srcs = select({
+        "@platforms//cpu:aarch64": [
+            "@libcap2-deb-aarch64//:lib/aarch64-linux-gnu/libcap.so.2",
+            "@libcap2-deb-aarch64//:lib/aarch64-linux-gnu/libcap.so.2.25",
+        ],
+        "//conditions:default": [
+            "@libcap2-deb//:lib/x86_64-linux-gnu/libcap.so.2",
+            "@libcap2-deb//:lib/x86_64-linux-gnu/libcap.so.2.25",
+        ],
+    }),
     hdrs = [
         "usr/include/sys/capability.h",
     ],
diff --git a/third_party/libseccomp2/BUILD b/third_party/libseccomp2/BUILD
index dd0c1848..bc9a2cc5 100644
--- a/third_party/libseccomp2/BUILD
+++ b/third_party/libseccomp2/BUILD
@@ -1,5 +1,8 @@
 alias(
     name = "libseccomp2",
-    actual = "@libseccomp2-dev-deb//:libseccomp2",
+    actual = select({
+        "@platforms//cpu:aarch64": "@libseccomp2-dev-deb-aarch64//:libseccomp2",
+        "//conditions:default": "@libseccomp2-dev-deb//:libseccomp2",
+    }),
     visibility = ["//visibility:public"],
 )
diff --git a/third_party/libseccomp2/libseccomp2.BUILD b/third_party/libseccomp2/libseccomp2.BUILD
index 1d6c5d9d..4ce426f5 100644
--- a/third_party/libseccomp2/libseccomp2.BUILD
+++ b/third_party/libseccomp2/libseccomp2.BUILD
@@ -1,9 +1,15 @@
 cc_library(
     name = "libseccomp2",
-    srcs = [
-        "@libseccomp2-deb//:lib/x86_64-linux-gnu/libseccomp.so.2",
-        "@libseccomp2-deb//:lib/x86_64-linux-gnu/libseccomp.so.2.5.1",
-    ],
+    srcs = select({
+        "@platforms//cpu:aarch64": [
+            "@libseccomp2-deb-aarch64//:usr/lib/aarch64-linux-gnu/libseccomp.so.2",
+            "@libseccomp2-deb-aarch64//:usr/lib/aarch64-linux-gnu/libseccomp.so.2.5.3",
+        ],
+        "//conditions:default": [
+            "@libseccomp2-deb//:lib/x86_64-linux-gnu/libseccomp.so.2",
+            "@libseccomp2-deb//:lib/x86_64-linux-gnu/libseccomp.so.2.5.1",
+        ],
+    }),
     hdrs = [
         "usr/include/seccomp.h",
         "usr/include/seccomp-syscalls.h",

