diff --git a/score/language/futurecpp/tests/member_iterator_test.cpp b/score/language/futurecpp/tests/member_iterator_test.cpp
index bea1361..6d649d2 100644
--- a/score/language/futurecpp/tests/member_iterator_test.cpp
+++ b/score/language/futurecpp/tests/member_iterator_test.cpp
@@ -9,6 +9,7 @@
 #include <forward_list>
 #include <list>
 #include <vector>
+#include <algorithm>
 
 #include <gtest/gtest.h>
 
diff --git a/score/language/futurecpp/tests/multi_span_test.cpp b/score/language/futurecpp/tests/multi_span_test.cpp
index 9b57903..c0da98b 100644
--- a/score/language/futurecpp/tests/multi_span_test.cpp
+++ b/score/language/futurecpp/tests/multi_span_test.cpp
@@ -11,6 +11,7 @@
 #include <cstdint>
 #include <numeric>
 #include <sstream>
+#include <algorithm>
 
 #include <gtest/gtest.h>
 
diff --git a/score/mw/log/detail/logging_identifier.h b/score/mw/log/detail/logging_identifier.h
index ec02bac..b15bdfa 100644
--- a/score/mw/log/detail/logging_identifier.h
+++ b/score/mw/log/detail/logging_identifier.h
@@ -19,6 +19,7 @@
 
 #include <array>
 #include <functional>
+#include <algorithm>
 
 namespace score
 {
